packages/react-devtools-extensions/chrome/manifest.json

```diff
@@ -59,7 +61,8 @@
       "js": [
         "build/prepareInjection.js"
       ],
-      "run_at": "document_start"
+      "run_at": "document_start",
+      "all_frames": true
```

Comment:
Can you actually share a code sandbox, where I could test and check which cases this would solve?